deployer/hub.py

```diff
@@ -5,347 +5,22 @@
 import subprocess
 import sys
 import tempfile
-from contextlib import contextmanager, redirect_stderr, redirect_stdout
+import pytest
```

Comment:
PEP8 is a style guide describing how to format imports that we don't enforce but like following. It does help me I think - separating the std. lib imports from non-std lib imports and local imports --- that's a great convention i think.

You typically don't have a line separator between import and from statements, but put the from statements after the direct imports.

> Imports should be grouped in the following order:
>
>1. Standard library imports.
>2. Related third party imports.
>3. Local application/library specific imports.
>
>You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports